submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/model/Submission.kt

```diff
@@ -91,6 +91,9 @@ class Submission(
     @Column
     var secretKey: String = ""
 
+    @Column
+    var status: String = ""
```

Comment:
I think we can use here a `@Converter` and an enum type to avoid setting the wrong value.

Reply:
Done